spec/models/note_spec.rb

```diff
@@ -14,4 +14,116 @@
   it 'has a valid factory' do
     expect(subject).to be_valid
   end
+
+  describe '.content_length' do
+    let(:north_utility) { create(:north_utility, code: 1) }
+    let(:south_utility) { create(:south_utility, code: 2) }
+
+    let(:north_user) { create(:user, utility_id: north_utility.id) }
+    let(:south_user) { create(:user, utility_id: south_utility.id) }
```

Comment:
¡Muy bien! Pero te propongo unos pequeños cambios sobre como ordenar las declaraciones.

Nosotros queremos testear notas con validaciones de una utility X
Luego separamos los tests por cada una de ellas, por ejemplo: "Cuando es North..." o "Cuando es South..."

¿Qué te parece si acá simplemente creamos un `user` y una `note` cuyo contenido es variable? No nos cambia saber de quién es el usuario ni de quien es la nota en realidad.

```suggestion
    let(:user) { create(:user, utility: utility) }
    let(:note) { create(:note, user: user, content: content, note_type: :critique) }
```